api/sheets/get_name.md

```diff
@@ -1,46 +1,72 @@
 
-## General information about this formula
+> **Note**
+> 
+> Be sure to follow the instructions for [Installing and Enabling the Sheets Add-On](/api/sheets/) before using this formula.
 
-**Formula**: `=DCGETNAME(dcids)`
+## Formula
 
-**Required arguments**:
+```
+=DCGETNAME(dcids)
+```
 
-* `dcids` - A list of nodes to query, identified by their DCIDs.
+## Required Arguments
+
+`dcids` - A list of DCID nodes to query
+
+## Returns
+
+The names associated with given DCIDs to a cell or a column range of cells.
+
+> **Note**:
+> It’s best to minimize the number of function calls to `=DCGETNAME(dcids)` by using a single call to get the names for a column of nodes. This is because a spreadsheet will make one call to a Google server [*per custom function call*](https://developers.google.com/apps-script/guides/sheets/functions#optimization). If your sheet contains many thousands of separate calls to `=DCGETNAME(dcids)` you can expect it to be slow and return with errors.
 
 ## Examples
 
-Before trying this method out, make sure to follow the setup directions in [the main section for Sheets docs](/api/sheets/index.html).
+This section contains examples of returning the names associated with given DCIDs.
+
+### Example 1: Retrieve the Name of a Country by its DCID
 
-### Example 1: Retrieve the name of a country by its DCID.
+To retrieve the name of a country by its DCID, perform the following steps:
+
+1. Place your cursor in the cell where you want to add a DCID. In this case, cell A1.
+2. Click in the search box for the **Fill place dcids for selected cells** section. For this example, enter “Ivory Coast”. The cell is populated with a DCID of “country/CIV”.
+3. Move to the cell where you want to retrieve the country name.
+4. Enter the formula **`=DCGETNAME(A1)`** to retrieve the country name and press **Enter**.  The French spelling of Ivory Coast (Côte d'Ivoire) populates the cell.
 
 ![](/assets/images/sheets/sheets_get_name_cote_d_ivoire_cropped.png)
 
-### Example 2: Retrieve the names of a collection of cell lines.
+### Example 2: Retrieve the Names of a Collection of Cells
 
-![](/assets/images/sheets/sheets_get_name_cell_lines_cropped.png)
+To retrieve the names of countries by a collection of cells, perform the following steps:
 
->  **TIP:**
->
->  It's best to minimize the number of function calls to `DCGETNAME` by using a single call to get names for a column of nodes. This is because a spreadsheet will make one call to a Google server [per custom function call](https://developers.google.com/apps-script/guides/sheets/functions#optimization). If your sheet contains thousands of separate calls to `DCGETNAME`, expect it to be slow and with errors.
+1. Enter into column A the DCIDs that are shown in the following image.
+2. In cell B1, enter the formula `=DCGETNAME(A1:A4)`.
+3. Press **Enter** and the names of the countries for each DCID populate in column B.
 
+![](/assets/images/sheets/sheets_get_name_cell_lines_cropped.png)
```

Comment:
The image for this now has country names, not cell lines.

Reply:
I'm not completely clear on the edit you want me to make. Should I just rename the image file?